CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+* v1.6.0
```

Comment:
Can replace the 1.5.1 item if we decide to release only 1.6.0, but not 1.5.1 before. Please advise.

Reply:
I suggest to do bugfixes in patches and features in minors ;)

Reply:
well, if a single release does both bugfixes and features, it is fine. Forcing to split into 2 releases just to have only features in 1.6.0 does not make sense.

Reply:
ah, only one line is actively supported here

Reply:
1.6.0 looks good to me.